bot/exts/christmas/advent_of_code/_cog.py

```diff
@@ -0,0 +1,365 @@
+import asyncio
+import json
+import logging
+import math
+from datetime import datetime, timedelta
+from pathlib import Path
+from typing import Tuple
+
+import discord
+from discord.ext import commands
+from pytz import timezone
+
+from bot.bot import Bot
+from bot.constants import (
+    AdventOfCode as AocConfig, Channels, Colours, Emojis, Month, Roles, WHITELISTED_CHANNELS,
+)
+from bot.exts.christmas.advent_of_code import _helpers
+from bot.utils.decorators import in_month, override_in_channel, with_role
+
+log = logging.getLogger(__name__)
+
+AOC_REQUEST_HEADER = {"user-agent": "PythonDiscord AoC Event Bot"}
+
+EST = timezone("EST")
+COUNTDOWN_STEP = 60 * 5
+
+AOC_WHITELIST = WHITELISTED_CHANNELS + (Channels.advent_of_code,)
+
+
+def is_in_advent() -> bool:
+    """Utility function to check if we are between December 1st and December 25th."""
+    # Run the code from the 1st to the 24th
```

Comment:
These two comments don't match (24th and 25th).

Reply:
This is part of the old, existing code. I'd rather focus on getting the new feature merged before trying to polish the older parts. We simply don't have time; we need the new features ASAP.

Reply:
I've changed the docstring to reflect it